### PR TITLE
tests: move some helper functions to util

### PR DESCRIPTION
**Pull request by overvenus**
Move some helper functions and structs to mod `util`, so that we can reuse them in failpoints tests, eg. #2639

### CODE REVIEW

**Comment by siddontang** [comment]
Please add a description.

**Comment by huachaohuang** [comment]
LGTM

File: tests/raftstore/transport_simulate.rs [hunk 1]
```diff
@@ -637,3 +637,25 @@ impl Clone for RandomLatencyFilter {
         }
     }
 }
+
+#[derive(Clone, Default)]
+pub struct LeaseReadFilter {
```
Comment (BusyJay) [inline tests/raftstore/transport_simulate.rs]: I prefer keeping these in the original places. They are only used in one case, keep them close makes it easier to read the case.

Comment (overvenus) [reply tests/raftstore/transport_simulate.rs]: I am going to use it in #2639 .

**Comment by siddontang** [comment]
LGTM

**Comment by hicqu** [comment]
LGTM.
BTW, how to use methods from it? `use tests::util::some_methods` or `use tikv::tests::util::some_methods`?

**Comment by overvenus** [comment]
`use raftstore::util::*` does the job, it's in tests scope, and here the `tikv` is an extern crate, see more in `integrations.rs`.

**Pull request merged**